test/configs/zombieTanssi.json

```diff
@@ -83,7 +83,8 @@
         "collators":  [
           {
             "name": "DummyCollator-2001",
-            "command": "../target/release/container-chain-template-frontier-node"
+            "command": "../target/release/container-chain-template-frontier-node",
+            "ws_port": 9951
```

Comment:
I used the same port number in my PR, could you change it?

Reply:
OFC